pkg/hyperstart/types.go

```diff
@@ -207,7 +207,7 @@ type Container struct {
 	Rootfs           string              `json:"rootfs"`
 	Fstype           string              `json:"fstype,omitempty"`
 	Image            string              `json:"image"`
-	Addr             string              `json:"addr,omitempty"`
+	SCSIAddr         string              `json:"scsiAddr,omitempty"`
```

Comment:
Why do you want to change the name ? `Addr` is generic enough to include SCSI case , right ?

Reply:
I had this as Addr in the agent, but changed this to SCSIAddr following your review comment which I agree as SCSIAddr makes it clearer. Hence the change here.

Reply:
I have forgotten...